src/bp/common/http.ts

```diff
@@ -13,6 +13,16 @@ const escapeHtmlSimple = (str: string) => {
     .replace(/`/g, '&#96;')
 }
 
+export const URL_FOLDER_SEPERATOR = '|!|slash|!|'
+
+export const encodeFolderPath = (path: string): string => {
+  return path.replace('/', URL_FOLDER_SEPERATOR)
```

Comment:
this will only replace the first occurence in the string. I think there could be more than one level down the road

Reply:
Other modules might try to do api requests to foldered flows without changing their `/` with `|!|slash|!|`

Reply:
some bots in production have folders so multiple / ...

Reply:
Good catch @EFF just pushed a fix where I switched to a regex, multilevel folders now work